File: codewar/7-kyu/square_digits.php
<?php
// https://www.codewars.com/kata/546e2562b03326a88e000020/train/php

function square_digits(int $num)
{
    return (int)implode("", array_map(fn ($n) => $n * $n, str_split($num)));
}

print_r(gettype(square_digits(9119) ). "\n"); // 811181


$aa = 123;
$dd = str_split($aa);
print_r($dd);

File: codewar/V1/Day-146/square_digits.py
# https://www.codewars.com/kata/546e2562b03326a88e000020/train/python


def square_digits(num):
    str_n = list(str(num))
    for number in str_n:
        # print(int(number) ** 2, end="")
        res += int(number) ** 2 + ""
    return res


x = 9119
print(square_digits(x))
